web/packages/teleterm/package.json

```diff
@@ -53,7 +53,7 @@
     "react-dnd": "^14.0.4",
     "react-dnd-html5-backend": "^14.0.2",
     "whatwg-url": "^14.2.0",
-    "zod": "^3.25.67",
+    "zod": "^4.1.4",
```

Comment:
zod ships with a built-in way to convert a schema to a JSON schema.

https://zod.dev/v4?id=json-schema-conversion